lib/Model/ShareWrapper.php

```diff
@@ -541,12 +570,43 @@ public function importFromDatabase(array $data, string $prefix = ''): IQueryRow
 		return $this;
 	}
 
+
+	/**
+	 * Load from database format (JSON string) to IAttributes
+	 * based on \OC\Share20\DefaultShareProvider
+	 */
+	private function importAttributesFromDatabase(string $data): void {
+		if ($data === '') {
+			return;
+		}
+
+		$attributes = new ShareAttributes();
+		$compressedAttributes = json_decode($data, true);
+		if (!is_array($compressedAttributes)) {
+			return;
+		}
+
+		foreach ($compressedAttributes as $compressedAttribute) {
+			$attributes->setAttribute(
```

Comment:
Shorter version: 
```suggestion
			$attributes->setAttribute(...$compressedAttribute);
```
A bit different if other arguments are available…